specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/VpnGatewayGet.json

```diff
@@ -28,15 +28,58 @@
                 "remoteVpnSite": {
                   "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/vpnSites/vpnSite1"
                 },
-                "connectionStatus": "Connected",
+                "enableInternetSecurity": false,
                 "ingressBytesTransferred": 0,
                 "egressBytesTransferred": 0,
-                "routingWeight": 0,
-                "connectionBandwidth": 100,
-                "sharedKey": "key",
-                "enableBgp": false,
-                "useLocalAzureIpAddress": false,
-                "ipsecPolicies": []
+                "vpnLinkConnections": [
+                  {
+                    "name": "Connection-Link1",
+                    "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/vpnGateways/gateway1/vpnConnections/vpnConnection1/VpnSiteLinkConnections/Connection-Link1",
+                    "etag": "w/\\00000000-0000-0000-0000-000000000000\\",
+                    "properties": {
+                      "provisioningState": "Succeeded",
+                      "vpnSiteLink": {
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/vpnSites/vpnSite1/vpnSiteLinks/siteLink1"
+                      },
+                      "connectionBandwidth": 200,
+                      "ipsecPolicies": [],
+                      "vpnConnectionProtocolType": "IKEv2",
+                      "sharedKey": "key",
+                      "ingressBytesTransferred": 0,
+                      "egressBytesTransferred": 0,
+                      "enableBgp": false,
+                      "enableRateLimiting": false,
+                      "useLocalAzureIpAddress": false,
+                      "usePolicyBasedTrafficSelectors": false,
+                      "routingWeight": 0
+                    },
+                    "type": "Microsoft.Network/vpnGateways/vpnConnections/VpnSiteLinkConnections"
```

Comment:
`type` is not present in specs. Please make examples to correlate with real server responses and the adjust specs to match with them.
Please check the same in all Get/List operations.

Reply:
type is referred using ./network.json#/definitions/Resource